### ISSUE TITLE
Test failing on CI: RubEditingAreaTest>>testEmbeddedAnchorInTextAreDrawnAtRightPosition

### ISSUE THREAD

**Issue by MarcusDenker** [guillep/pharo-core#10625]
**Bug description**
RubEditingAreaTest>>testEmbeddedAnchorInTextAreDrawnAtRightPosition started failing on the CI for all platforms.

maybe related to merging https://github.com/pharo-project/pharo/pull/10558 ?

**Comment by kasperosterbye** [comment]
They all (all RubEditingAreaTest ) run on P10 build: 340, commit: fa6ca09, date: 8 December 2021.
They were green yesterday.
But it seem plausible that the merge causes the problems as the method was not present before the merge. There was a problem in that it opened some morphs on the (non-existing) screen, but I just checked the actual code in the repository, which is the same as in what I downloaded just now.

Is FormCanvas defined in headless mode?
